packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -53,7 +53,7 @@ export const SendModalView = ({ closeModal }: Props) => {
             fieldValue,
             wallet.erc20TokenBalanceSelected.decimals,
           );
-          const userBalance = wallet.erc20TokenBalanceSelected.amount;
+          const userBalance = wallet.erc20TokenBalanceSelected.spendableAmount;
```

Comment:
Bug fix, because user can't spend more than spendable amount. This is the real user balance available to spend.